api/controller/websiteControler.js

```diff
@@ -0,0 +1,36 @@
+const validator = require('../validate.js');
+var Message = require('../models/websiteModel.js');
+var myexpress = require("express")
+var router = myexpress.Router()
+module.exports = {
+
+    createWebsite:async (req, res) => {
+        try {
+          const validationRule = {
+            "cpu": [
+              "required",
+              "wedCpuTypes"
+            ],
+            "title": "required|string|min:3|max:50|EnglishLetters",
+            "description": "required|string|min:10|max:100|desEnglishLetters",
+            "domain":"isDomainAvailable",
+            "typeOfDomain":"tapedomin"
+          };
+          await validator(req.body, validationRule, {}, (err, status) => {
```

Comment:
dont send req.body. use a different variable for more readability.
const website = req.body for example.

Reply:
שינתי